### PR TITLE
Treeview: Fixed focus bug (404) and  state using CSS content being read bug (406) 

### PR DESCRIPTION
**Pull request by jongund**

### CODE REVIEW

File: examples/treeview/treeview-2/css/treeLinks.css [hunk 1]
```diff
@@ -23,29 +23,39 @@ ul[role="tree"] a {
...
+[role="treeitem"][aria-expanded="false"] > span > span.state {
   color: #630;
   font-weight: bold;
   font-size: 1em;
+  position: relative;
+  top: -0.1em;
+}
+
+[role="treeitem"][aria-expanded="false"] > span > span.state:before {
+  content: "\00229E \0000A0";
```
Comment (mcking65) [inline examples/treeview/treeview-2/css/treeLinks.css]: Still here; still causing issue 406

File: examples/treeview/treeview-2/treeview-2a.html [hunk 2]
```diff
@@ -55,7 +55,7 @@ <h3 id="tree1">Foods</h3>
           <ul role="tree" aria-labelledby="tree1" >
             <li role="treeitem"
                 aria-expanded="false">
-              <span>Fruits</span>
+              <span><span class="state" aria-hidden="true"></span>Fruits</span>
```
Comment (mcking65) [inline examples/treeview/treeview-2/treeview-2a.html]: the aria-hidden will not apply to a pseudo element

**Comment by jongund** [comment]
Matt,

I reset my fork of the practices and made the following changes to the treeview examples:


  *   Removed default keyboard focus style from treeitems
  *   Changed the use of character entities to images in treeview-2 example

Looks like my pull request should now be clean.

Jon

**Comment by mcking65** [comment]
Replaced by pr #411. Closing.

**Pull request closed without merging**